package.json

```diff
@@ -3,12 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.12.4",
+    "@material-ui/lab": "^4.0.0-alpha.61",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "array-to-tree": "^3.3.2",
     "bootstrap": "4.6.0",
     "js-cookie": "^2.2.1",
+    "material-ui": "^0.20.2",
```

Comment:
![image](https://user-images.githubusercontent.com/18077542/176165722-7858f441-975c-4581-bf87-0b9ab9d1522b.png)

However, recommended package is already included. Do we still need it?

Reply:
Removed the package.. working fine without it